@stellar/design-system/src/components/Layout/index.tsx

```diff
@@ -65,7 +66,12 @@ const Header: React.FC<HeaderProps> = ({
   menu,
   contentCenter,
   contentRight,
+  showTitle,
 }: HeaderProps) => {
+  // In order to be able to save unique settings per project in ToggleDarkMode,
+  // we need to require a projectTitle but in some cases we don't need to display it.
+  const _projectTitle = showTitle === true ? projectTitle : undefined;
```

Comment:
I'd like to just take an optional `projectTitle` also in `Layout.Header` but we need a unique string per project to support the `ToggleDarkMode` functionality. Is there a better way to do this?